src/main/java/org/springframework/data/tarantool/core/mapping/TarantoolIdClass.java

```diff
@@ -0,0 +1,45 @@
+package org.springframework.data.tarantool.core.mapping;
+
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.data.annotation.Persistent;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Specifies a composite primary key for this entity (marked as {@link Tuple}.
+ *
+ * The specified class must contain all properties
+ * that are parts of primary index of the tuple.
+ * These properties must be present in the entity
```

Comment:
If we consider marking the properties with `@Id` not mandatory, do not forget to change this sentence